data/human/transport missions.txt

```diff
@@ -3209,3 +3209,92 @@ mission "Earth Retirement"
 			label reject
 			`	Charles looks surprised for a moment, and then, for the first time you've seen, he smiles. "Thank you, Captain <last>. I wish there were more people like you in the Paradise Worlds."`
 			`	You wish the couple the best of luck on Earth, and they slowly make their way to the train station. Presumably, their new apartment is far from here.`
+
+
+
+mission "Spacediving professionals"
+	name "Spacediving at <planet>"
+	description "Head to destination with <bunks> skydivers, then drop them from the edge of space as you land. They are wearing thermal spacesuits, durable parachutes and oxygen bottles to ensure their survival. Payment is <payment>. This type of job has risks of non-payment and fines."
```

Comment:
> This type of job has risks of non-payment and fines

How would you know this in advance? This kind of text amounts to giving the game away immediately.

Reply:
I want to position the skydiving jobs as "higher payment on average, but also higher risk of non-payment or even fines" and I'm searching for the amount of "reasonable upfront warning" vs the amount of "just find out as you are playing".

Reply:
Maybe I could focus on "just find out as you are playing" where the "negative" jobs only start showing after 10 "good" jobs.

Reply:
I choose this last strategy (with the negative jobs starting to show some time after the positive ones).